src/styles/blocks/ingridients.scss

```diff
@@ -0,0 +1,104 @@
+.ingridients {
+  &__title {
+    @extend %text-h2;
+    font-size: 24px;
```

Comment:
Replace the hard-coded font-size values with SCSS variables for better maintainability. E.g. $font-size-h2-mobile, $font-size-h2-tablet, and $font-size-h2-desktop.